insights/core/__init__.py

```diff
@@ -1542,3 +1542,150 @@ def __init__(self, *args, **kwargs):
         deprecated(AttributeDict, "Please set attributes explicitly.")
         super(AttributeDict, self).__init__(*args, **kwargs)
         self.__dict__ = self
+
+
+class ModInfo(Parser):
```

Comment:
You should use `CommandParser` as the base class

Reply:
Done